ui/src/components/LeftNav/LeftNav.vue

```diff
@@ -1,80 +1,118 @@
 <template>
-  <div id="left-nav">
-    <h2><span v-on:click="showAllMaps">Maps</span> <span v-if="activeMap">/ {{activeMap.name}}</span></h2>
+  <div id="left-nav" class="sidebar" :class="sidebartoggle">
+    <div class="toggle" @click="togglesidebar">
+      <svg
+        class="toggle-arrow"
+        width="2em"
+        height="2em"
+        viewBox="0 0 16 16"
+        fill="currentColor"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          fill-rule="evenodd"
+          d="M5.854 4.646a.5.5 0 0 0-.708 0l-3 3a.5.5 0 0 0 0 .708l3 3a.5.5 0 0 0 .708-.708L3.207 8l2.647-2.646a.5.5 0 0 0 0-.708z"
+        />
+        <path
+          fill-rule="evenodd"
+          d="M10 8a.5.5 0 0 0-.5-.5H3a.5.5 0 0 0 0 1h6.5A.5.5 0 0 0 10 8zm2.5 6a.5.5 0 0 1-.5-.5v-11a.5.5 0 0 1 1 0v11a.5.5 0 0 1-.5.5z"
+        />
+      </svg>
+    </div>
+    <h2>
+      <span v-on:click="showAllMaps">Maps</span>
+      <span v-if="activeMap">/ {{ activeMap.name }}</span>
+    </h2>
     <div class="container">
-      <ul id="maps-list" v-if="!activeMap" >
+      <ul id="maps-list" v-if="!activeMap">
         <MapRow
           v-for="map in capabilities.maps"
           v-bind:key="map.name"
           v-bind:map="map"
         />
       </ul>
       <ul id="map-layers-list" v-if="activeMap && mapIsReady">
-        <MapLayerRow 
+        <MapLayerRow
           v-for="layer in activeMap.layers"
           v-bind:key="layer.name"
           v-bind:layer="layer"
         />
       </ul>
     </div>
     <div id="left-nav-footer" v-if="activeMap">
-      <div class="btn"
-        v-on:click="toggleFeatureInspector"  
-        v-bind:class="{active: inspectorIsActive}"><span class="dot"></span>Inspect Features
+      <div
+        class="btn"
+        v-on:click="toggleFeatureInspector"
+        v-bind:class="{ active: inspectorIsActive }"
+      >
+        <span class="dot"></span>Inspect Features
       </div>
     </div>
   </div>
 </template>
 
 <script>
-import MapRow from './MapRow.vue'
-import MapLayerRow from './MapLayerRow.vue'
+import MapRow from "./MapRow.vue";
+import MapLayerRow from "./MapLayerRow.vue";
 import { store, mutations } from "@/globals/store";
 import { map } from "@/globals/map";
 
-const mapboxgl = require('mapboxgl');
+// const mapboxgl = require('mapboxgl');
+import mapboxgl from "mapbox-gl";
 
 export default {
-  name: 'LeftNav',
-  components:{
+  name: "LeftNav",
+  components: {
     MapRow,
-    MapLayerRow
+    MapLayerRow,
   },
   props: {
-    capabilities: Object
+    capabilities: Object,
   },
-  data(){
+  data() {
     return {
       inspectorIsActive: false,
-      inspector: null
-    }
+      inspector: null,
+      sidebartoggle: "",
```

Comment:
maybe name this `sidebarToggleClass` to indicate what value it's holding. Also maybe return the default value here.